### DISCUSSION TITLE
How to enable duplication in IsarLinks?

### DISCUSSION THREAD

**Discussion by fearschism** [isar/isar#1061] [Help] [answered]
I want to enable IsarLinks to have the same duplicated id in the collection.

example:
 [A,B,C] -> [A,A,B,C] <-can contain the same collection twice

Thank you.

**Comment by joneldominic** [comment]
Can you give an example use case related to this? It's probably not a good practice to have duplicated ids.

**Comment by fearschism** [comment, marked as answer]
I Apologise for the late response,

I had two schemas called Task and another one called Resource.

 the use-case requirement, A task can have multiple resources even if it's duplicated.

Resources -> [a,b,c]

What I expected.
Task A {
...
IsarLinks<Resource> Resources {
Resource A,
Resource A(again),
Resource C
}
}

solution:
Task A {
...
List<int> repeatedIDs = {
1,1,2
}

IsarLinks<Resource> Resources {
Resource A,
Resource C
}
}

So, I managed to solve it by adding a column of repeated ids to calculate repeated resources in a task.

Thank you.
